dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -18,7 +18,8 @@ RUN apt-get update &&   \
         rrdtool         \
         python-smbus    \
         ethtool         \
-        dmidecode
+        dmidecode &&    \
+    pip install enum34
```

Comment:
It doesn't appear this is being used. Am I missing something?

Reply:
Nevermind. I see this is used in https://github.com/Azure/sonic-platform-daemons/pull/52.